inst/shiny/modules/tab_nca/setup.R

```diff
@@ -0,0 +1,141 @@
+#' NCA Setup Server Module
+#'
+#' This module handles the server-side logic for the NCA settings, including file uploads,
+#' analyte/dose/specimen selection, AUC intervals, NCA parameters, as well as slope selection and
+#' their unit specification. Then it integrates them to adnca_data object and returns data
+#' that is processed accordingly.
+#'
+#' @param id ID of the module.
+#' @param data Reactive with data table containing raw data uploaded to the app.
+#' @param adnca_data Reactive with `PKNCAdata` object including the `adnca_data`.
+#'
+#' @returns List with three reactive expressions:
+#'   * processed_pknca_data - PKNCAdata object with applied settings and slope selections.
+#'   * settings - List with raw settings as gathered by the module.
+#'   * slope_rules - Data frame with slope inclusions / exclusions provided by slope selector.
+
+setup_ui <- function(id) {
+  ns <- NS(id)
+
+  navset_pill_list(
+    nav_panel(
+      "Settings",
+      fluidRow(
+        column(6,
+          fileInput(
+            ns("settings_upload"),
+            width = "100%",
+            label = "Upload settings",
+            buttonLabel = list(icon("folder"), "Browse"),
+            accept = c(".csv", ".xpt")
```

Comment:
issue: put the expected format
```suggestion
            accept = ".rds"
```